WordPress/src/main/java/org/wordpress/android/viewmodel/activitylog/ActivityLogViewModel.kt

```diff
@@ -182,15 +188,29 @@ class ActivityLogViewModel @Inject constructor(
         }
     }
 
+    // todo: annmarie - Remove once the feature exclusively uses the more menu
     fun onActionButtonClicked(item: ActivityLogListItem) {
         if (item is Event) {
-            if (backupFeatureConfig.isEnabled()) {
-                // todo: annmarie implement expand/dismiss the more menu
-                _showMoreMenu.value = item
-            } else {
-                _showRewindDialog.value = item
+            _showRewindDialog.value = item
+        }
+    }
+
+    fun onSecondaryActionClicked(
+        secondaryAction: ActivityLogListItem.SecondaryAction,
+        item: ActivityLogListItem
+    ): Boolean {
+        if (item is Event) {
+            val navigationEvent = when (secondaryAction) {
+                RESTORE -> {
+                    ShowRestore(item)
+                }
+                DOWNLOAD_BACKUP -> {
+                    ShowBackupDownload(item)
+                }
             }
+            _navigationEvents.value = org.wordpress.android.viewmodel.Event(navigationEvent)
```

Comment:
Minor (🔍 ): Since this `Event` import seems to be more generic, consider moving this import up and use `ActivityLogListItem.Event`everywhere instead. This way, the whole `org.wordpress.android.viewmodel.Event` will be avoided. Also, the usage of `ActivityLogListItem.Event` will not be confused with the generic view model `Event`. Wdyt?

Reply:
@ParaskP7 - I was dreaming out this last night. I am so changing this.

Reply:
Such sweet dreams you have Annmarie! 🤣